### DISCUSSION TITLE
How about nubia?

### DISCUSSION THREAD

**Discussion by cedriclemercier** [zenfyrdev/bootloader-unlock-wall-of-shame#59] [General]
It seems they were under ZTE in the past but I am curious of the state of red magic phones

**Comment by BotchedRPR** [comment]
Terrible. Cannot unlock without a leak of an engineering bootloader

**Reply by MegadronA03** [reply]
Why this information is not on ZTE/nubia page?

**Reply by Lost-Entrepreneur439** [reply]
No one has provided a source for it, and I don't really know anything about RedMagic, they don't exist where I am (ZTE only really sells budget devices here)
